go.mod

```diff
@@ -4,6 +4,9 @@ go 1.21
 
 require (
 	github.com/BurntSushi/toml v1.3.2
+	github.com/aws/aws-sdk-go-v2 v1.25.3
```

Comment:
I'd prefer if we could use `github.com/minio/minio-go/v7` instead. The S3 api is the standard API for blob storage in the cloud so with `minio-go` we can support multiple backends.